resources/lib/token_decoder.py

```diff
@@ -0,0 +1,79 @@
+import re
+import math
+
+
+# the + in the beginning parses text to a number

```

Comment:
the javascript version still writes a 'reqkey':'oiheroie' into document.cookie with the correct value